config/jobs/kubernetes/sig-scalability/sig-scalability-experimental-periodic-jobs.yaml

```diff
@@ -443,3 +443,45 @@ periodics:
           - --test-cmd-name=ClusterLoaderV2
           - --timeout=40m
           - --use-logexporter
+
+  # cluster volume load scaling by number of nodes
+- name: ci-kubernetes-storage-scalability-cluster-load-persistent-volume-500-node
+  tags:
+    - "perfDashPrefix: storage-cluster-load-persistent-volume-500-node"
+    - "perfDashJobType: storage"
+  interval: 1h
+  labels:
+    preset-service-account: "true"
+    preset-k8s-ssh: "true"
+    preset-e2e-scalability-node: "true"
+  annotations:
+    testgrid-dashboards: sig-scalability-experiments
+    testgrid-tab-name: cluster-load-persistent-volume-500-node
+  spec:
+    containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20190726-d319b3f-master
+        args:
+          - --repo=k8s.io/kubernetes=master
+          - --repo=k8s.io/perf-tests=master
+          - --root=/go/src
+          - --timeout=60
+          - --scenario=kubernetes_e2e
+          - --
+          - --check-leaked-resources
+          - --cluster=
+          - --extract=ci/latest
+          - --gcp-node-image=gci
+          - --gcp-nodes=500
```

Comment:
This won't work for regular projects.
See my comment here: https://github.com/kubernetes/test-infra/pull/13583#discussion_r307287714

Reply:
Is it possible to have this test start running before https://github.com/kubernetes/perf-tests/pull/677 is implemented? Sorry to rush you but my internship is going to be over mid August and it would be nice to see some more interesting tests running by then. 

If not, @verult will hopefully be taking over.

Reply:
we need to double check with PD folks before we enable this since this is going to be creating a lot of pds and generating a lot of operations

Reply:
Our 1 node tests already generated alerts from the GCE backend team. I can add this to the issue.